agents/negamax.c

```diff
@@ -6,12 +6,15 @@
 
 #include "game.h"
 #include "negamax.h"
+#include "zobrist.h"
 
 #define MAX_SEARCH_DEPTH 6
 
 static int history_score_sum[N_GRIDS];
 static int history_count[N_GRIDS];
 
+static unsigned long long hash_value;
```

Comment:
Shall we explicitly use `uint64_t` for portability?

Reply:
Yes, it would be safer to use uint64_t. I will propose a patch for this.